### DISCUSSION TITLE
getting an error in lesson 7 at 16:21.

### DISCUSSION THREAD

**Discussion by KAravind2005** [Cyfrin/foundry-full-course-cu#898] [Q&A] [answered]
error,

Error: 
Failed to resolve file: "/forge-std/Test.sol": No such file or directory (os error 2).
 Check configured remappings..
    --> "/home/aravind/solidity-course/foundry-f23/foundry-fund-me-f23/test/FundMeTest.t.sol"
        "/forge-std/Test.sol"

**Comment by theirrationalone** [comment, marked as answer]
Please remove that very first slash from this: "/forge-std/Test.sol"

If you're using right import statement like this:

```solidity
import {Test} from "forge-std/Test.sol";
```

and still getting that same error then Please restart your vs-code.
